tests/test_generate.py

```diff
@@ -133,3 +135,50 @@ def test_generate(yaml_dir, tmpdir):
     html_blob = bucket.get_blob(html_blob_name)
     t5 = html_blob.updated
     assert t4 == t5
+
+
+class TestGenerate(unittest.TestCase):
+    def test_format_docfx_json(self):
+        self.maxDiff = None
+        metadata = metadata_pb2.Metadata()
+        metadata.name = "doc-pipeline"
+        metadata.xrefs.extend(["one.xml", "two.xml"])
```

Comment:
Not that it really matters, but the xref files are yml rather than xml.

Reply:
Fixed.